connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -544,6 +555,95 @@ public int generation() {
         return generation;
     }
 
+    @Override
+    public synchronized void getConfigDef(final String connType, final Callback<ConfigDefInfo> callback) {
```

Comment:
These two methods don't look like they should differ between `StandaloneHerder` and `DistributedHerder`. Can we move the implementation to `AbstractHerder`?